infra/gravity/cluster_status.go

```diff
@@ -23,16 +24,16 @@ func (c *TestContext) Status(nodes []Gravity) error {
 	}
 
 	err := retry.Do(ctx, func() error {
-		errs := make(chan error, len(nodes))
+		errors := make(chan error, len(nodes))
```

Comment:
Just a personal curiosity, err is the conventional variable name for an err, so is errs not the conventional name for a list or channel of errors?